uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-balance-tab.component.js

```diff
@@ -101,15 +106,17 @@ define([
      * @return {Promise}
      */
     function loadReportCurrentPage () {
+      var params = {
+        period_id: filters.absence_period,
+        type_id: filters.absence_type
+      };
+
+      (userRole === 'manager') && (params.managed_by = vm.loggedInContactId);
```

Comment:
I know this is simple JS logic, but it's too clever. Keep it simple, use an if:
```js
if (userRole === 'manager') {
  params.managed_by = vm.loggedInContactId;
}
```

Reply:
@AkA84 loves this syntax. I started liking it too. I will change, np.

Reply:
@reneolivo funnily enough, this change doesn't affect .min files!